library/borgbase.py

```diff
@@ -140,8 +176,16 @@ def main():
     else:
         key_id = get_key_id(module.params['ssh_key'])
 
-    # Add new repo using the key
-    res = add_repo(key_id)
+    # Check if repo with given name exist
+    repo_exist, repo_id = get_repo_id(module.params['repository_name'])
+
+    if repo_exist == True:
```

Comment:
`if repo_exist:` is enough and easier to read.

Reply:
done